File: week12/SacramentPlanner/SacramentPlanner/ClientApp/src/app/talk.model.ts
import { Member } from "./members/member.model";

export class Talk {
  public talkTitle: string
  public speaker: Member
  public timeLimit: number

  constructor(speaker: Member = null, timeLimit: number = 0, talkTitle: string = "") {
    this.talkTitle = talkTitle
    this.speaker = speaker ? speaker : new Member()
    this.timeLimit = timeLimit
  }
}

File: week12/SacramentPlanner/SacramentPlanner/Models/TalkModel.cs
using System;
namespace SacramentPlanner.Models
{
    public class TalkModel
    {
        public string TalkTitle { get; set; }
        public MemberModel Speaker { get; set; }
        public int TimeLimit { get; set; }

        public TalkModel(string talkTitle, MemberModel speaker, int timeLimit)
        {
            TimeLimit = timeLimit;
            Speaker = speaker;
            TalkTitle = talkTitle;
        }

        public TalkModel(){}
    }
}
